user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -80,8 +81,53 @@ bool Timepix3TrigEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::
 
 uint64_t Timepix3RawEvent2StdEventConverter::m_syncTime(0);
 bool Timepix3RawEvent2StdEventConverter::m_clearedHeader(false);
+bool Timepix3RawEvent2StdEventConverter::applyCalibration(false);
+bool Timepix3RawEvent2StdEventConverter::first_time(true);
+std::string Timepix3RawEvent2StdEventConverter::calibrateDetector("");
+std::string Timepix3RawEvent2StdEventConverter::calibrationPath("");
+std::string Timepix3RawEvent2StdEventConverter::threshold("");
+std::vector<std::vector<float>> Timepix3RawEvent2StdEventConverter::vtot;
+std::vector<std::vector<float>> Timepix3RawEvent2StdEventConverter::vtoa;
+
 bool Timepix3RawEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::StandardEventSP d2, eudaq::ConfigurationSPC conf) const{
 
+  // Read from configuration:
+  if(first_time) {
```

Comment:
I'm cheating with the first_time bool here because I didn't yet find out yet how to implement something like an Initialize() function here. Obviously, I don't want to read in the data file each time I'm processing a new event, i.e. for every single pixel hit.

Reply:
@simonspa What's your opinion on this? Should we add an `Initialize()` function the the converters or use this workaround here?

Reply:
We cannot because of the framework design. The way these converters are instantiated doesn't allow for it because we will never have a stable object that is re-used, but it might be reinstantiated as any time again anew. Or one instance might be used by different detectors etc.

Not nice, and we need all of these annoying `static` workarounds but that's the only way.

Reply:
I understand, then we'll have to live with this. Thanks for the explanation.